src/iter/mod.rs

```diff
@@ -85,13 +85,26 @@ mod unzip;
 #[cfg(test)]
 mod test;
 
+/// `IntoParallelIterator` implements the conversion to a [`ParallelIterator`].
+/// By implementing `IntoParallelIterator` for a type, you define how it will
+/// transformed into an iterator. This is a parallel version of the standard
+/// library's `std::iter::IntoIterator` trait.
```

Comment:
We could make `std::iter::IntoIterator` link to https://doc.rust-lang.org/std/iter/trait.IntoIterator.html

Reply:
@KodrAus I added that link as well